pkg/kubernetes/scripts/virtual-machines/utils.jsx

```diff
@@ -26,3 +28,21 @@ export function getPairs(object) {
         value: object[key]
     }))
 }
+
+export function vmIdPrefx(vm) {
+  return `vm-${vm.metadata.name}`
+}
+
+// TODO: set log level, i.e. reuse window.debug
+export function logDebug(...args) {
+  console.log('Kubevirt: ', ...args);
```

Comment:
Could this use `console.debug()` for now, then?

Reply:
sure, good point